dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/validation/XMLValidator.java

```diff
@@ -34,6 +36,15 @@ private XMLValidator() {}
     /** {@inheritDoc} */
     @Override
     public void validate(ValidationHelper helper, Context context, String key, XML t) {
-        // TODO Auto-generated method stub
+        if (t != null) {
+            if (StringUtils.isNotBlank(t.getNamespace())) {
+                String namespace = t.getNamespace();
+                String object = "XML";
+                if (!ValidatorUtils.isValidURL(namespace)) {
```

Comment:
namespace is a URI (different from URL). Create a method similar to isValidURL in ValidatorUtils - to check for URI instead and should catch URISyntaxException instead.